server/server.js

```diff
@@ -13,23 +12,25 @@ app.use(express.static('public'));
 
 app.get('/api/:ticker', (req, res) => {
   const ticker = path.basename(req.url);
-  console.log(ticker);
   const body = {};
   const userId = Math.floor(Math.random() * (21 - 1) + 1);
 
   Users.findOne({ where: { id: userId }, raw: true })
     .then((result) => body.user = result)
     .then(() => Stocks.findOne({ where: { ticker }, raw: true }))
     .then((result) => {
-      body.stock = result;
-      res.send(body);
+      if (!result) {
+        res.status(400);
```

Comment:
small but a response 400 is a bad request. i think you want to send a 404 for not found.

Reply:
Good call, I'll change that.